var/exp_repo/experiments/scaling/experiment.py

```diff
@@ -0,0 +1,65 @@
+from benchpark.directives import variant
+
+
+class Scaling(object):
```

Comment:
I would call this `ScalingExperiment` instead of just `Scaling`, I think it's a bit clearer that way